src/yunohost/utils/error.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+""" License
+
+    Copyright (C) 2018 YUNOHOST.ORG
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program; if not, see http://www.gnu.org/licenses
+
+"""
+
+from moulinette.core import MoulinetteError
+from moulinette import m18n
+
+class YunohostError(MoulinetteError):
+    """Yunohost base exception"""
+    def __init__(self, key, __raw_msg__=False, *args, **kwargs):
+        if __raw_msg__:
```

Comment:
Should be `raw_msg` and not `__raw_msg__`, dunderscores are only used for privates methodes.

Also I think that `m18n.n` will return the raw string if no translations are found so I'm not sure that this is needed.